terraform/environments/staging/portal.tf

```diff
@@ -355,7 +355,7 @@ locals {
     # Sign Up
     {
       name  = "SIGN_UP_WHITELISTED_DOMAINS"
-      value = "firezone.dev,firez.one,pentesters.oneleet.com"
+      value = "firezone.dev,firez.one,firezonedemo.com"
```

Comment:
Is `firezonedemo.com` a new domain for us?

Reply:
Nah we have a few that we use for non-production things

Reply:
gotcha